cdk/lib/__snapshots__/content-api-firehose-client-testing.test.ts.snap

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The ContentApiFirehoseClientTesting stack matches the snapshot 1`] = `
+{
+  "Metadata": {
+    "gu:cdk:constructs": [
+      "GuAllowPolicy",
+      "GuGithubActionsRole",
+    ],
+    "gu:cdk:version": "TEST",
+  },
+  "Outputs": {
+    "GithubActionsRoleGithubActionsRoleArnC13D9654": {
+      "Value": {
+        "Fn::GetAtt": [
+          "GithubActionsRoleF5CC769F",
+          "Arn",
+        ],
+      },
+    },
+  },
+  "Resources": {
+    "GithubActionsOidc": {
+      "Properties": {
+        "ClientIdList": [
+          "sts.amazonaws.com",
+        ],
+        "ThumbprintList": [
+          "6938fd4d98bab03faadb97b34396831e3780aea1",
+          "1c58a3a8518e8759bf075b76b750d4f2df264fcd",
+        ],
+        "Url": "https://token.actions.githubusercontent.com",
+      },
+      "Type": "AWS::IAM::OIDCProvider",
+    },
+    "GithubActionsRoleF5CC769F": {
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Statement": [
+            {
+              "Action": "sts:AssumeRoleWithWebIdentity",
+              "Condition": {
+                "StringLike": {
+                  "token.actions.githubusercontent.com:sub": "repo:guardian/content-api-firehose-client:*",
+                },
+              },
+              "Effect": "Allow",
+              "Principal": {
+                "Federated": {
+                  "Ref": "GithubActionsOidc",
+                },
+              },
+            },
+          ],
+          "Version": "2012-10-17",
+        },
+        "Tags": [
+          {
+            "Key": "gu:cdk:version",
+            "Value": "TEST",
+          },
+          {
+            "Key": "gu:repo",
+            "Value": "guardian/content-api-firehose-client",
+          },
+          {
+            "Key": "Stack",
+            "Value": "content-api-firehose-client",
+          },
+          {
+            "Key": "Stage",
+            "Value": "TEST",
+          },
+        ],
+      },
+      "Type": "AWS::IAM::Role",
+    },
+    "kinesisstreamaccessFE21B9E2": {
+      "Properties": {
+        "PolicyDocument": {
+          "Statement": [
+            {
+              "Action": "kinesis:DescribeStreamSummary",
+              "Effect": "Allow",
+              "Resource": "arn:aws:kinesis:undefined:undefined:stream/content-api-firehose-v2-CODE",
```

Comment:
@devx-reliability-and-ops - we also get `undefined:undefined` when we run `npm run synth` - are there some flags we should add to get the correct account number and region output?

Reply:
I've added a comment against the offending line in `cdk/lib/content-api-firehose-client-testing.ts`.